tests/benchmarks/numeric/vanZadelhoff_1_3D_mesher.py

```diff
@@ -91,6 +91,7 @@ def nTT (r):
     model.geometry.points.position.set(mesh.points)
     model.geometry.points.velocity.set(np.zeros((npoints, 3)))
 
+    model.geometry.points.multiscale.set_all_neighbors(n_nbs,nbs)
```

Comment:
Perhaps, move to `magritte.setup.set_Delaunay_neighbor_lists` cfr. previous comment.